[Functorch] Delete unused files

malfet opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #83464
* __->__ #83777
* #84788

Differential Revision: [D39478381](https://our.internmc.facebook.com/intern/diff/D39478381)

@diff-train-skip-merge Manually copied this commit to fbsync to fix shipit

malfet committed:
[Functorch] Delete unused files

[ghstack-poisoned]

malfet committed:
Update on "[Functorch] Delete unused files"




[ghstack-poisoned]

zou3519 commented on functorch/.github/workflows/docs.yml:
```diff
@@ -1,82 +0,0 @@
-name: Build and Deploy Docs
-on:
-  pull_request:
-    types: [opened, synchronize, reopened]
-  push:
-    branches:
-      - main
```
We still need the docs workflow; we have not migrated it over to PyTorch yet

zou3519 replied:
Actually, let me think about this for a little more (it depends on what the next PR up in the stack is doing)

malfet replied:
Well, I don't think it works that way, is it? (Unless this folder is still mirrored to github.com/pytorch/functorch)

zou3519 replied:
>  (Unless this folder is still mirrored to github.com/pytorch/functorch)

It is, for now at least

malfet committed:
Update on "[Functorch] Delete unused files"




[ghstack-poisoned]

malfet committed:
Update on "[Functorch] Delete unused files"




[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D39032967).

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D39032967).

botmethere commented:
@pytorchbot label ciflow/trunk

malfet committed:
Update on "[Functorch] Delete unused files"


Differential Revision: [D39032967](https://our.internmc.facebook.com/intern/diff/D39032967)

[ghstack-poisoned]

malfet committed:
Update on "[Functorch] Delete unused files"


Differential Revision: [D39032967](https://our.internmc.facebook.com/intern/diff/D39032967)

[ghstack-poisoned]

malfet committed:
Update on "[Functorch] Delete unused files"


Differential Revision: [D39032967](https://our.internmc.facebook.com/intern/diff/D39032967)

[ghstack-poisoned]

zou3519 reviewed (approved):
LGTM, but please wait until we disable shipit for pytorch/functorch to merge this

malfet committed:
Update on "[Functorch] Delete unused files"


Differential Revision: [D39032967](https://our.internmc.facebook.com/intern/diff/D39032967)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D39058811).

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D39032967).

malfet commented:
@pytorchbot merge -g

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D39478381).
